packages/core/src/domain/internalMonitoring/index.ts

```diff
@@ -0,0 +1,14 @@
+export {
+  InternalMonitoring,
+  MonitoringMessage,
+  monitored,
+  monitor,
+  callMonitored,
+  addMonitoringMessage,
+  addMonitoringError,
+  startFakeInternalMonitoring,
+  resetInternalMonitoring,
+  setDebugMode,
+  startInternalMonitoring,
+} from './internalMonitoring'
+export { startMonitoringBatch } from './startMonitoringBatch'
```

Comment:
I would be in favor of this index file. It allows to separate the implementation into different sub-modules but keeping `imports` from other modules short and focused.

Reply:
I am not against it but:
- should we start do that everywhere?
- do we need to explicitly list all the exported members or could we go with `*`?

Reply:
> should we start do that everywhere?

* If the folder is grouping modules that are mostly used together and expose a coherent API together accessed by other parts of the SDK. This can be seen as a privacy convention at the modules level.

* If the folder is grouping similar but mostly indepentent modules (for example, `src/domain`), I don't think it's worth it.

> do we need to explicitly list all the exported members or could we go with *?

The advantage of explicit exports is to chose what should be public and what should be kept private at the module level (ex: for spec files). Again, this is a convention, but it feels natural to me. 

IMO `startMonitoringBatch` should not be exported here, because it is only used by sibling modules.

Reply:
Yes could be nice to have a consistent approach here. I also, think that being able to import everything from everywhere is not great. Even if that's just a convention, I'd say let's discuss about it ;)